src/SdkCommon/ClientRuntime.Azure/ClientRuntime.Azure/Properties/AssemblyInfo.cs

```diff
@@ -9,7 +9,7 @@
 [assembly: AssemblyTitle("Microsoft Rest Azure Client Runtime")]
 [assembly: AssemblyDescription("Client infrastructure for Azure client libraries.")]
 [assembly: AssemblyVersion("3.0.0.0")]
-[assembly: AssemblyFileVersion("3.3.12.0")]
+[assembly: AssemblyFileVersion("3.3.13.0")]
```

Comment:
@j5lim can you revert this change. The version number will be updated once this is ready to be published.